rest/data/pod_test.go

```diff
@@ -53,6 +57,26 @@ func (s *podConnectorSuite) TestCreatePod() {
 	s.Require().NoError(err)
 	s.Assert().Equal("secret", podDB.Secret)
 	s.Assert().Equal("env_value", podDB.TaskContainerCreationOpts.EnvVars["env_name"])
+	s.NoError(db.Clear(pod.Collection))
```

Comment:
isn't this already accomplished by `TeardownTest()`?

Reply:
Fixed. I think this was left over from when I rebased to fix merge conflicts.